README.md

```diff
@@ -78,7 +78,7 @@ Example:
 ![example](Images/on_token_needed.PNG)
 
 ### Set Token Using Key and Secret
-Similar to [Set Token](#set-token) except it takes an app key and secret as parameters and automatically generates a token. **Do not use this function in production and do not allow your app key and secret to leak.**
+For convenience during early development and prototyping, this function is provided to acquire a client access token directly from within the application. However, please note **we do not recommend** using this mechanism in the production software for [security best practices](https://docs.dolby.io/communications-apis/docs/guides-client-authentication). App secret needs to be protected and not included in the application.
```

Comment:
I would keep the first sentence of the original description as well, at least the part about this function being similar to Set Token as it has the same effect in terms of initializing the plugin.